src/css/profile/mobile/themes/light.variables.less

```diff
@@ -26,6 +26,8 @@
     @feature-card-underline-color: #eaeaea;
     @feature-card-subtitle-color: #43a6b0;
     @flicker-pointer-color: #ABD7EB;
+
+    @text-color: #252525;
```

Comment:
this property is already defined in `base.less`

Reply:
Question)
Why we define colors for the light theme to the base.less?
Is it default value for the light.theme?
Can I define it to the light.variable.less?

Reply:
Is there any reasons to define the "--text-color" to the :root ?
Can I remove the "@text-color" and --text-color in base.less?

Reply:
Good question. 
The `base.less` was supposed to contain common values.
`light.less` and `dark.less` was supposed to contain differences.
You can move `@text-color` to light theme if think that it will be better but try don't duplicate variables.

Reply:
IMO, values that do not change depending on the theme shoud be in base.less.
Such as "primary-color".
So I think moving to "oneui-common.less" looks good because "text-color" changes depding on the theme.

Reply:
There are a lot of values that are defined incorrectly, I will try to re-arrange them later.